src/core/api/user.go

```diff
@@ -339,6 +341,57 @@ func (ua *UserAPI) ToggleUserAdminRole() {
 	}
 }
 
+// ListUserPermissions handles GET to /api/users/{}/permissions
+func (ua *UserAPI) ListUserPermissions() {
+	if ua.userID != ua.currentUserID {
+		log.Errorf("Current user, id: %d can not view other user's permissions", ua.currentUserID)
```

Comment:
Use debug rather than error to log the message, or this will make users confuse when debugging the system.

Reply:
changed to `Warningf`, which is used by `ToggleUserAdminRole` method